web/src/index.html

```diff
@@ -3,8 +3,11 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-    <title>CadHub</title> <!-- to be replaced by react helmet -->
+    <meta name="keywords" content="cadhub, app, application, web application, cad, code, code-cad, cad-modeling, community, website, cascade studio, openscad" />
+    <meta property="og:type" content="website" />
+    <meta name="robots" content="index, follow" />
+    <link rel="icon" type="image/png" href="/favi2.jpg" />
```

Comment:
I don't think this <link> should be here as the favicon is set as a webpack plugin.
https://github.com/Irev-Dev/cadhub/blob/main/web/config/webpack.config.js
Could we remove it?

Reply:
right, I will remove this line